.github/workflows/pull_request_label.yml

```
@@ -6,7 +6,7 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v3
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        sync-labels: true
+      - uses: actions/labeler@v4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Since quotes are removed from expressions.

```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```